index.js

```diff
@@ -21,6 +23,9 @@ const OPERATIONS = {
 }
 
 const defaultOptions = {
+  tempRetryTimes: 5,
+  tempRetryInterval: 50,
+  retryTemporaryErrors: true,
```

Comment:
Makes sense. I was on the fence for that; I'll update this (and the usage PRs) tomorrow